### ISSUE TITLE
Uintah Mailing List - Messages Undeliverable

### ISSUE THREAD

**Issue by gruneisen** [Uintah/Uintah#2]
Hello -

I know that this is not an issue with Uintah itself, but I have been attempting to send messages to the users mailing list and have gotten everything bounced back to me as un-deliverable. I have registered my email addresses with the user's group and verified they are on the list.

Is there another step I need to complete before being able to use the user's list?

Thank you!

**Comment by QuocAnh90** [comment]
I got the same problem.

**Comment by tharman-git** [comment]
Gruneisen,

We are undergoing a transition with our email lists.
Please open a new issue and post your Uintah question or issue.

Todd
